updater/bundle-updater.ts

```diff
@@ -0,0 +1,95 @@
+import * as request from 'request';
```

Comment:
Can you switch to 'node-fetch'? I'm trying to get rid of 'request' library

Reply:
I'll refactor it to node-fetch once your PR adding it is merged, which I think we can probably go ahead and merge now.